roles/oidc-playground-client/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: create apache folder
+  file:
+    path: "/var/www/oidc-playground"
+    owner: apache
```

Comment:
owner should be root, we don't want to make the web server able to write in this directory

Reply:
Done